src/qt/forms/receiverequestdialog.ui

```diff
@@ -115,7 +280,7 @@
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
-        <set>QDialogButtonBox::Close</set>
```

Comment:
`Close` seems more correct.

Reply:
From  [`QDialogButtonBox` docs](https://doc.qt.io/qt-5/qdialogbuttonbox.html#StandardButton-enum):
- An "OK" button defined with the `AcceptRole`.
- A "Close" button defined with the `RejectRole`.

It seems the `AcceptRole` is more appropriate for interaction with this dialog.
Also see @promag's [comment](https://github.com/bitcoin/bitcoin/pull/17600#issuecomment-558392202).